### ISSUE TITLE
Alarm doesn't work

### ISSUE THREAD

**Issue by sargoon** [renyuneyun/Easer#232]
**Describe the bug**
When I set the alarm, the alarm does not work at the time I set.

**To Reproduce**
1. Start the app.
2. Go to 'Profile' section.
3. Click on '+' button.
4. Create a profile with selecting "Set Alarm" option.
5. Set an alarm and you will see alarm doesn't work.

**Expected behavior**

**Extra phone info (please complete the following information):**
- ROM: MIUI Global 10.3.6
-  Android Version: Android 9.0

**Recording Of The Bug**
![DeadForthrightDarwinsfox-size_restricted](https://user-images.githubusercontent.com/34961616/60584622-649a4d80-9d96-11e9-872c-64ef5b6e9956.gif)

**Comment by racuna** [comment]
Maybe you should set the conditions to trigger that profile (e.g. day of the week), and then link your profile with the condition using a Script: e.g. if (day of the week)-> set (alarm at 8am)

**Comment by sargoon** [comment]
> 
> 
> Maybe you should set the conditions to trigger that profile (e.g. day of the week), and then link your profile with the condition using a Script: e.g. if (day of the week)-> set (alarm at 8am)

Hi there, thanks for the reply. Please check: https://github.com/renyuneyun/Easer/issues/233

**Comment by sargoon** [comment]
Hi @renyuneyun what are your thoughts about this bug? I am looking forward to your comment.
I've also reported this: https://github.com/renyuneyun/Easer/issues/233

**Comment by renyuneyun** [comment]
As in #233, probably you could re-try and see if it works?

But basically, if you really want to test a specific Profile, you can long press on its name, and trigger it manually.

**Comment by sargoon** [comment]
Yes, I tried it but the alarm still doesn't work, I can provide a gif if you want.

**Comment by renyuneyun** [comment]
@sargoon Yes, please :)
